### ISSUE TITLE
Sharable iframe

### ISSUE THREAD

**Issue by pwjablonski** [popcodeorg/popcode#1473]
I've noticed a lot of online editors have the feature to share with/embed with iframe. Would be a nice to have as we start getting popcode on more blog posts etc...!

**Comment by outoftime** [comment]
If I may play skeptic on this one, I’m wary of motivating features with “other [superficially] similar code editors do it”. A more relevant question is, how would this feature facilitate learning and/or support classroom workflows?

This particular feature I think is very closely tied to the use case of most Popcode-like editors out there, namely sharing code samples among professionals. In that context it certainly makes sense that you’d want to embed your code sample in the blog post you wrote about it. But it’s not clear how this feature would support Popcode’s goals. Maybe I am missing it though!

**Comment by pwjablonski** [comment]
Fair point! I mostly wanted to log the thought somewhere. I agreed on it doesn't directly support popcodes main goals, though I could see the rare instances in which it would be nice to have.

**Comment by outoftime** [comment]
Can you elaborate? Definitely down to keep this open (but deprioritized) if we can identify educational use cases : )

**Comment by ashleykolodziej** [comment]
I'd like to be able to embed a Popcode editor into a page on my LMS - an iframe would be perfect for that. I want my students to be able to try to write and edit code while they are reading the tutorial step by step.

Embeds of gists or existing code would be great too so that they can start from any point on the page, or check their answers above against my sample code below.
